client/elements/suttaplex/sc-suttaplex-list.html

```diff
@@ -238,6 +238,10 @@
               return this.suttaplexReturns ? (this.suttaplexReturns.length - index) : 0;
           }
 
+          _calculateOpened(suttaplexReturns) {
+              return (suttaplexReturns.length <= 3) ? true : false;
```

Comment:
the ternary operator is redundant. `(suttaplexReturns.length <= 3)` already returns only two possible values - `true` or `false`.